### ISSUE TITLE
Restrict Access to App Based on Group Membership

### ISSUE THREAD

**Issue by davidwozabal** [nextcloud/mail#5997]
### Feature Request

Restrict access to app to specific user groups.

#### Summary

It would be great, if it would be possible to restrict access to the mail app to specific user groups. Ideally the icon of the app in the app bar would only appear for those groups that have access to the app.

**Comment by jesmh** [comment]
That exists already. 
Go to → Users and create a group e.g. 'mail'. Go to → Apps search for Mail find Mail and click on the Mail app to see its details. Here you find the option 'limit to groups'. Enable it and in the dropdown choose the group you just created under Users. You can find the group by simply typing while cursor is in the dropdown.
Hope that helps

**Comment by GretaD** [comment]
Yes, what @jesmh said is correct. They refer to "more details" to the sidebar on the right, that opens after you click the Mail App on the App list.

**Comment by GretaD** [comment]
Closing as the feature requested already exists

**Comment by davidwozabal** [comment]
Thanks a lot - that worked perfectly!
